### PR TITLE
Add new package: py-shakemap

### PR DESCRIPTION
**Pull request by darmac**

### CODE REVIEW

File: var/spack/repos/builtin/packages/py-shakemap/package.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    url      = "https://github.com/usgs/shakemap/archive/4.0.2.tar.gz"
+
+    version('4.0.2', sha256='52a7f640d4ff98a99703d7d9564fcf3437d0cd23cac5e6ba23d88b17db6c580d')
+    version('4.0.1', sha256='fa1a7a994aaa1217b3a3892acabdc4f3ccd15500abd0b0b4449e85ab8312681b')
+    version('4.0.0', sha256='65e73c56a09239c0fa83f31f430ccae983bb0748e25f98ad67a22d0f26e0b50d')
+    version('0.91',  sha256='548a64c0433225217625e85c9321a0df3f0b586fb6e0ced2767731d2f9349468')
+    version('0.9',   sha256='8de233d3dfd3ef94e6680805a265f6c8845540c0fb2b5671ede64de97ec47a33')
+
+    depends_on('py-numpy', type=('build', 'run'))
+    depends_on('py-cython', type=('build', 'run'))
```
Comment (alalazo) [inline var/spack/repos/builtin/packages/py-shakemap/package.py:24]: Ugh, it's quite difficult to say if the list of deps is correct looking through [here](https://github.com/usgs/shakemap/blob/master/install.sh). Can you please double check or confirm you can install this on some platform?

**Review by alalazo — COMMENTED** [review]
LGTM, modulo one minor request

File: var/spack/repos/builtin/packages/py-shakemap/package.py (hunk 1 above)
Comment (adamjstewart) [reply var/spack/repos/builtin/packages/py-shakemap/package.py:24]: Also, the cython dependency is likely only needed at build-time.

Comment (adamjstewart) [reply var/spack/repos/builtin/packages/py-shakemap/package.py:24]: Ping @darmac

**Comment by alalazo** [comment]
Closing as stale. Feel free to reopen if you want to push this PR forward.

**Pull request closed without merging**